src/libraries/System.Text.Json/tests/JsonNumberTestData.cs

```diff
@@ -2,11 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Buffers;
 using System.Collections.Generic;
 using System.Globalization;
-using System.IO;
-using Newtonsoft.Json;
```

Comment:
nit: Remove unrelated cleanup like this (especially in a file that would otherwise not be touched by your PR).